### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by AlexForJava**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -21,3 +21,5 @@
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+*.iml
```
Comment (Taniann) [inline .gitignore:24]: that file shouldn't be in homework PR

File: src/com/gmail/chernii/oleksii/Main.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+ * Created by Space on 07.03.2019.
+ */
+public class Main {
+    private static Calculator calculator = (first, second, action) ->
+            (action == '+') ? (first + second) :
+                    (action == '*') ? (first * second) :
+                            (action == '-') ? (first - second) :
+                                    (action == '$' && second != 0) ? (Math.pow(first, 1 / second)) :
+                                            (action == '^') ? (Math.pow(first, second)) :
+                                                    (action == '#' && first != 0) ? ((first + second) / first * 1.0 + 117.) : 0;
```
Comment (Taniann) [inline src/com/gmail/chernii/oleksii/Main.java]: better make user informed that there is no such action instead of returning 0

**Review by Taniann — CHANGES_REQUESTED** [review]
good
but add case with dividing

**Comment by AlexForJava** [comment]
Sorry, I forget about .gitignore

Thank you 

8 марта 2019, 22:37:59, от "Алексей" <hqq@ukr.net>:

@Taniann requested changes on this pull request.

good
but add case with dividing

In .gitignore:

> @@ -21,3 +21,5 @@
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+*.iml
that file shouldn't be in homework PR

In src/com/gmail/chernii/oleksii/Main.java:

> @@ -0,0 +1,32 @@
+package com.gmail.chernii.oleksii;
+
+import java.util.Scanner;
+
+/**
+ * Created by Space on 07.03.2019.
+ */
+public class Main {
+    private static Calculator calculator = (first, second, action) ->
+            (action == '+') ? (first + second) :
+                    (action == '*') ? (first * second) :
+                            (action == '-') ? (first - second) :
+                                    (action == '$' && second != 0) ? (Math.pow(first, 1 / second)) :
+                                            (action == '^') ? (Math.pow(first, second)) :
+                                                    (action == '#' && first != 0) ? ((first + second) / first * 1.0 + 117.) : 0;
better make user informed that there is no such action instead of returning 0

**Pull request closed without merging**